grasping_tp/GraspServerRequest request
---
grasping_tp/GraspConfigList grasp_configs
